instagram/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+
+import './App.css';
+import CommentSection from './components/CommentSection/commentSection';
+import dummyData from './dummy-data';
+import PostContainer from './components/PostContainer/postContainer';
+import SearchBar from './components/SearchBar/searchBar';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state ={ 
+      dummyData:[],
```

Comment:
don't name this dummyData, it is the live data in your state, regardless of where it comes from originally